test/smoke/smoke-tests.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+const testConfig = require('./config');
+const testURL = `${testConfig.url}/details/404`;
+
+describe('Smoke Tests', () => {
+
+  before(() => {
+    require('../acceptance/mixins/login')(browser);
+    browser.shouldBeOn404Page = () => {
+      browser.getText('main').should.contain('Not found');
+    };
+  });
+
+  describe('Trying to see a non-existent record', () => {
```

Comment:
`when I try to view a non-existent record` ? (Just so the English reads nicely in the logs.)